spinta/formats/ascii/commands.py

```diff
@@ -47,18 +47,30 @@ def _render(
     status_code,
     headers,
 ):
-    # Format params ar given in RQL query `?format(width(1),colwidth(1))`.
+    # Format params ar given in RQL query `?format(width(1),max_col_width(1))`.
     width = params.formatparams.get('width')
-    colwidth = params.formatparams.get('colwidth')
+    max_col_width = params.formatparams.get('max_col_width')
+    max_value_length = params.formatparams.get('max_value_length', 100)
+    rows_to_read = params.formatparams.get('rows_to_read', 200)
```

Comment:
This parameter should not be exposed to api, because some bad actors could use this for memory attacks.